app/_locales/en/messages.json

```diff
@@ -1324,6 +1327,9 @@
   "learmMoreAboutGas": {
     "message": "Want to $1 about gas?"
   },
+  "learnCancelSpeeedup": {
+    "message": "Learn how to $1"
```

Comment:
same here. (I realize the one above it doesn't have the description but we should try and include where we can!)